recipes/libbsc/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "libbsc" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IlyaGrebnov/libbsc/archive/{{ version }}.tar.gz
+  sha256: 6645d95d929ffe9c5adcdbbdcd7151aa122356640ef4ece90d5b62e3d48fd007
+  patches:
+    - latest_github_version_plus_multithreading.patch
```

Comment:
After doing the change above, you can add your changes here.

Reply:
I meant only your changes.

Reply:
I made the suggested updates and pushed the code